docs/faq.md

```diff
@@ -394,15 +395,13 @@ When you run remote code, you often need to install packages (e.g. scipy) or hav
 
 ### <a name="benchmark-cli"></a> How can I benchmark different methods?
 
-The most flexible way to do so is to write a custom remote launcher script. This
```

Comment:
again this is a better answer imo. Im judging it from the lens of a first time user (Im one of them)

Reply:
Changed it slightly. The stuff in benchmarking/benchmark_loop is outdated and will be removed. The up-2-date docs is the tutorial now.